user_input.py

```diff
@@ -0,0 +1,59 @@
+from textwrap import dedent
+from time import sleep
+from random import choice
+
+savage1 = {"ducky": "Has feathers and goes quack quack", "red": "The colour your face should be", "cup": "sippy sippy", "feeble": "An assessment of your mental prowess..."}
+
```

Comment:
As a whole the content of this file seems tightly coupled with game_engine in a confusing way- it has assumptions from game_engine (the number of options, the state of user-interaction) and its not got a clear domain. I'm assuming you haven't decided how you want to arrange this, so keep an eye on it.